dask-report.html

```diff
@@ -0,0 +1,88 @@
+
+
+
```

Comment:
thanks for the updates, @travisdriver. Looks like the `dask-report.html` file got checked in. Would you mind deleting it? Maybe we need to add it to the .gitignore?

Reply:
Sorry will delete it. Yes, let's add it to the .gitignore.